io/plugins/eu.esdihumboldt.hale.io.topojson/src/eu/esdihumboldt/hale/io/topojson/TopoJsonInstanceWriter.java

```diff
@@ -171,8 +177,60 @@ private void convertShapefileToTopoJson(String shapefilePath,
 			CoordinateReferenceSystem sourceCrs, String targetPath, String topologyName, int kink,
 			int quantizeDigit, boolean compressOutput) throws IOException {
 
-		TopojsonApi.shpToTopojsonFile(shapefilePath, sourceCrs, targetPath, topologyName, kink,
+		String aJson = TopojsonApi.shpToTopojson(shapefilePath, sourceCrs, topologyName, kink,
 				quantizeDigit, compressOutput);
+		try {
+			// Parse JSON string
+			ObjectMapper mapper = new ObjectMapper();
+			JsonNode jsonNode = mapper.readTree(aJson);
+
+			// Replace "\u0000" values with null recursively
+			replaceUnwantedCharacters(jsonNode);
+
+			// Convert modified JSON node back to string
+			String modifiedJsonString = mapper.writeValueAsString(jsonNode);
+
+			if (compressOutput) {
+				Toolbox.writeFile(targetPath, Compress.compressB64(modifiedJsonString));
+			}
+			else {
+				Toolbox.writeFile(targetPath, modifiedJsonString);
+			}
+
+		} catch (Exception e) {
+			e.printStackTrace();
+		}
+	}
+
+	/**
+	 * @param node to replace the unwanted characters
+	 * @return the clean node
+	 */
+	public static JsonNode replaceUnwantedCharacters(JsonNode node) {
+		if (node.isObject()) {
+			ObjectNode objectNode = (ObjectNode) node;
+			objectNode.fields().forEachRemaining(entry -> {
+				JsonNode value = entry.getValue();
+				if (value.isTextual() && value.textValue().contains("\u0000")) {
```

Comment:
What about the case with the null date?

Reply:
That's missing at the moment. 
I'm uncertain about the best approach to resolve the date issue.